Makefile

```diff
@@ -183,13 +183,13 @@ deps: vendor web/built
 
 build:
 	$(DOCKER_COMPOSE) pull --parallel --ignore-pull-failures
-	$(DOCKER_COMPOSE) build --force-rm
+	$(DOCKER_COMPOSE) build --force-rm --pull
```

Comment:
I just asked to romain the exact reason , it's because 
```
  	$(DOCKER_COMPOSE) pull --parallel --ignore-pull-failures 
```
pull only images and  --pull pull images requested in dockerfiles with from